api/src/pcapi/alembic/versions/20230327T124205_3acdce9ec195_drop_procuct_cgu_not_null.py

```diff
@@ -0,0 +1,19 @@
+"""drop_procuct_cgu_not_null
+"""
+from alembic import op
+
+
+# pre/post deployment: post
+# revision identifiers, used by Alembic.
+revision = "3acdce9ec195"
+down_revision = "aa37244acc4c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    op.execute('ALTER TABLE "product" DROP CONSTRAINT IF EXISTS "check_iscgucompatible_is_not_null";')
+
+
+def downgrade() -> None:
+    pass
```

Comment:
pas de downgrade ?

Reply:
Le message de commit ne le dit pas [1], mais cette contrainte est définie au niveau de la _table_. Elle fait doublon avec une contrainte `NOT NULL` définie au niveau de la _colonne_. Donc pas besoin de la restaurer en cas de downgrade, on n'en a vraiment pas besoin.

[1] Peut-être que ça pourrait faire l'objet d'un commit séparé pour plus de clarté ?